projects/plugins/jetpack/_inc/client/state/site/reducer.js

```diff
@@ -333,6 +334,31 @@ export function hasActiveScanPurchase( state ) {
 	return !! getActiveScanPurchase( state );
 }
 
+/**
+ * Return any active security bundles on the site
+ *
+ * @param {*} state - Global state tree
+ * @returns {object} A active security bundle on the site, undefined otherwise
+ */
+export function getActiveSecurityPurchase( state ) {
+	return find( getActiveSitePurchases( state ), purchase =>
+		isJetpackBundle( purchase.product_slug )
```

Comment:
I know `JETPACK_BUNDLES` only contains Security variants at the moment, but I find the use of this function slightly confusing given the name of the enclosing function.

Reply:
I'll take this opportunity to create an identical but separate function/ const for security. Conceptually we could introduce a bundle that breaks this logic.